.travis.yml

```diff
@@ -13,7 +13,7 @@ deploy:
   provider: script
   skip_cleanup: true
   branch: master
-  script: package_cloud push PronghornDigital/mtna/os/version ./*.rpm
+  script: package_cloud push PronghornDigital/mtna/fedora/23 ./*.rpm
```

Comment:
It's a PackageCloud.io required field. Usually different distros use different package managers, package types and package repositories. I don't think there is anything that is preventing this from being installed on other distros, especially since it's just the app. All of the Fedora specific stuff is in the server cookbook.